src/material/core/focus-indicators/_focus-indicators.scss

```diff
@@ -71,7 +72,7 @@
   .mat-focus-indicator.mat-option.mat-active::before,
 
   // The focus indicator in the calendar is placed on an element inside the cell.
-  .mat-calendar-body-active .mat-focus-indicator::before,
+  .mat-calendar-body-cell:focus .mat-focus-indicator::before,
```

Comment:
This is a bug fix caused by https://github.com/angular/components/pull/19304. What we had previously *should* work, unfortunately the class `.mat-calendar-body-active` is not removed from a cell when Shift+Tabbing from a cell. This means that if you focus on a cell, and shift tab away, the focus indicator remains. To fix this, we just render on the `:focus` pseudo-class instead.